dev-infrastructure/templates/svc-cluster.bicep

```diff
@@ -277,3 +280,22 @@ module acrPushRole '../modules/acr-permissions.bicep' = [
     }
   }
 ]
+
+resource clustersServiceAcrResourceGroups 'Microsoft.Resources/resourceGroups@2023-07-01' existing = [
```

Comment:
We could potentially combine `clustersServiceAcrResourceGroupNames` with `imageSyncAcrResourceGroupNames` if we expect that these will always be the same. I am not sure if this is true or not since we may have multiple ACRs